README.md

```diff
@@ -12,6 +12,7 @@ The goals of the wiki are explained in the [Manifest](MANIFEST.md) and contribut
   * [Data exchange](#data-exchange)
 - [Migration guidelines](#migration-guidelines)
 - [External ressources](#external-ressources)
+- [Pharo 8.0](#pharo-8.0)
```

Comment:
Here, I think that we do not need an additional Pharo 8 section, Refactoring should go with others and just say at the top of the page that everything described works for Pharo 8.0 and after.